data/blog/mobile-app-launch-profiling.md

```diff
@@ -0,0 +1,46 @@
+---
+title: 'Mobile App Launch Profiling'
+date: '2024-04-17'
+tags: ['mobile', 'ios', 'android', 'profiling']
+draft: false
+summary: "See what's happening in your app before your first line of code can even run."
+images: ['/images/mobile-app-launch-profiling/launch-profile.png']
+layout: PostLayout
+canonicalUrl: mobile-app-launch-profiling
+authors: ['andrewmcknight']
+---
+
+If you use Sentry’s Performance monitoring on iOS/macOS and Android, you have the ability to see [profiles](https://docs.sentry.io/product/profiling/) of how your app runs in the wild on real end user devices. Any time an automatic or manual transaction is recorded, that profiling data is attached. But are there times where you can’t record a transaction? 🤔
```

Comment:
```suggestion
If you use Sentry’s Performance monitoring on Cocoa and Android, you have the ability to see [profiles](https://docs.sentry.io/product/profiling/) of how your app runs in the wild on real end user devices. Any time an automatic or manual transaction is recorded, that profiling data is attached. But are there times where you can’t record a transaction? 🤔
```

Reply:
@bruno-garcia I'm actually going to revert this and go back to saying Cocoa app. The SDK is named sentry-cocoa, after all.